src/CarbonAware/src/Configuration/EmissionsDataCacheConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.Extensions.Configuration;
+
+namespace CarbonAware.Configuration;
+
+internal class EmissionsDataCacheConfiguration
+{
+    public const string Key = "EmissionsDataCache";
+
+    public bool Enabled { get; set; }
+
+    public int ExpirationMin { get; set; }
+
+    public void AssertValid()
+    {
+        if(ExpirationMin <= 0)
```

Comment:
Some tests still failed, so I think we can fix them with following change:

```suggestion
    public bool Enabled { get; set; } = false;

    public int ExpirationMin { get; set; } = 0;

    public void AssertValid()
    {
        if(Enabled && ExpirationMin <= 0)
```